packages/language-support/src/tests/autocompletion/databasenameCompletion.test.ts

```diff
@@ -111,6 +111,52 @@ describe('Can complete database names', () => {
     });
   });
 
+  test('Correctly completes database name even in a create alias statement', () => {
+    testCompletions({
+      query: 'CREATE ALIAS fo for DATABASE ',
+      dbSchema,
+      expected: [
+        { label: 'db1', kind: CompletionItemKind.Value },
+        { label: 'db2', kind: CompletionItemKind.Value },
+        { label: 'movies', kind: CompletionItemKind.Value },
+        { label: 'myMovies', kind: CompletionItemKind.Value },
+        { label: 'scoped.alias', kind: CompletionItemKind.Value },
+        { label: 'a.b.c.d', kind: CompletionItemKind.Value },
+        { label: '$param1', kind: CompletionItemKind.Variable },
+      ],
+      excluded: [
+        // validate invalid keyword bug isn't present
+        { label: '', kind: CompletionItemKind.Keyword },
+        // do not suggest non-string parameters
+        { label: '$param2', kind: CompletionItemKind.Variable },
+        { label: '$param3', kind: CompletionItemKind.Variable },
+      ],
+    });
+  });
+
+  test('Correctly completes database name even in a create alias statement including extra of spaces', () => {
+    testCompletions({
+      query: 'CREATE ALIAS foo          FOR     DATABASE ',
```

Comment:
I've added this extra test, which was the problematic bit in the previous solution